linked-list.rb

```diff
@@ -62,13 +62,35 @@ def display
        puts full_list.join("->")
    end
 
-   def include?(key)
+   def include?(value)
+     current = @head
+     while !(current.next_node == nil)
+       return true if current.value == value
+       current = current.next_node
+     end
+     return true if current.value == value
+     return false
    end
 
    def size
+     current = @head
+     size = 0
+     while !(current.next_node == nil)
+         size += 1
+         current = current.next_node
+     end
+     size += 1
    end
 
    def max
+     current = @head
+     max = current.value
+     while !(current.next_node == nil)
+       max = current.value if current.value > max
+       current = current.next_node
+     end
+     max = current.value if current.value > max
+    max
    end
```

Comment:
Mostly understand, but honestly can't completely remember this from two weeks ago. I'll review it again tonight. Thanks!